.dockerfiles/x86-64-unknown-linux-gnu/Dockerfile

```diff
@@ -4,7 +4,8 @@ ENV PATH "/root/.pony/ponyup/bin:$PATH"
 
 RUN apt-get update \
  && apt-get install -y \
-    curl
+    curl \
+    g++
```

Comment:
should this be `gcc`, since we don't require a C++ compiler?

Reply:
g++ brings all the required dependencies with it.

I know it works. I didn't test with gcc.

Reply:
Alright, this is fine for now